index.html

```diff
@@ -5,132 +5,179 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins:wght@100;300;400;700&display=swap" rel="stylesheet">
     <title>Mobile Version Skeleton</title>
 </head>
 <body>
-    
-<header>
-    <nav class="nav-bar"> 
-        
-        <div class="button-a">
-        <a href="" class="my-logo">My Logo</a>
-        </div>
-       
-        <div class="button-b">
-            <div class="burger-button"></div>
-        </div>
 
-    </nav>
-</header>
+    <header>
+        <nav class="nav-bar"> 
+            
+            <div class="button-a">
+            <a href="" class="my-logo">My Logo</a>
+            </div>
+           
+            <div class="button-b">
+                <div class="burger-button"></div>
+            </div>
+    
+        </nav>
+    </header>
 
 <section id="body">
+
     <div class="title">
         <h1>I'm Abel <br>Glad to see <br>you!</h1>
     </div>
+    
     <div class="text">
         <p>I am a software developer! I can help you build a product,  
             Take a look of my works. If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
     </div>
+    
     <div class="button-c">
         <a href="#" class="letsconnect">LET'S CONNECT</a>
     </div>
+    
     <div class="icons">
-        <a href="#" class="twitter"></a>
-        <a href="#" class="linkedin"></a>
-        <a href="#" class="github"></a>
+        <a href="#"><img src="./images/icons/twitter.png" alt="" class="icon"></a>
+        <a href="#"><img src="./images/icons/linkedin.png" alt="" class="icon"></a>
+        <a href="#"><img src="./images/icons/github.png" alt="" class="icon"></a>
+        <a href="#"><img src="./images/icons/medium.png" alt="" class="icon"></a>
+        <a href="#"><img src="./images/icons/v.png" alt="" class="icon"></a>
     </div>
 
 </section>
 <br> <br> <br>
-<section id="projects">
+
+<section id="projects1">
     <div class="image">
         <img src="./images/projects1.png" alt="">
     </div>
     <div class="project-title">
         <h2>Tonic</h2>
     </div>
     <div class="subtitle">
-        <p>CANOPY • Back End Dev • 2015</p>
+        <p>CANOPY <span class="dot">•</span> <span class="backend">Back End Dev</span> <span class="dot">•</span> <span class="num">2015</span></p>
     </div>
     <div class="project-text">
-        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <p>A daily selection of privately <br>personalized reads; no accounts or <br>sign-ups required.</p>
     </div>
-    <div class="lan-tags">
+    <div class="lan-tags">     
+    <div class="tag-button">
         <p class="html">html</p>
```

Comment:
- [ ] As mentioned before, this section is not necessary for this project so therefore it would be best to remove it 💯